lib/src/components/NxList/__tests__/NxList.test.tsx

```diff
@@ -4,192 +4,216 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import React, { ReactElement } from 'react';
-import { NxLoadError, NxLoadingSpinner } from '../../..';
+import React from 'react';
+import { within } from '@testing-library/react';
+import { rtlRender, rtlRenderElement, runTimers, userEvent } from '../../../__testutils__/rtlUtils';
+
 import NxList from '../NxList';
-import { NxListProps } from '../types';
-import { getShallowComponent } from '../../../__testutils__/enzymeUtils';
-import 'jest-enzyme';
-import { act } from 'react-dom/test-utils';
-import { mount, ReactWrapper } from 'enzyme';
-import NxListButtonItem from '../NxListButtonItem';
 
 describe('NxList', function() {
+  const quickRender = rtlRender(NxList, {}),
+      renderEl = rtlRenderElement(NxList, {});
 
-  let mountContainers: HTMLElement[] = [];
+  // prevent RTL logging thrown warnings
```

Comment:
Could you include a mention here of which warnings you're silencing?

Reply:
I was going to asking a question about this, there is a console warning for not having `emptyMessage` when no children, no error and not loading, but is this warning necessary if we already have default empty message `This list is empty.` when `emptyMessage` prop is not provided? 
`console.warn('emptyMessage is required when no list items are to be rendered');`

Reply:
> I was going to asking a question about this, there is a console warning for not having emptyMessage when no children, no error and not loading, but is this warning necessary if we already have default empty message This list is empty. when emptyMessage prop is not provided?
console.warn('emptyMessage is required when no list items are to be rendered');

I think that's ok.  From what I can tell/recall, the default empty message isn't really supposed to be used but is there as a sort of safeguard.  So keep your code as-is but add the comment saying which message you're silencing

Reply:
Add comments here 536f555b49bb126863e05f2f2142febad218a267